admin/config.js

```diff
@@ -21,6 +23,8 @@ module.exports = {
   SESSION_SECRET: process.env.NODE_ENV === 'production' ? process.env.SESSION_SECRET : 'anti tamper for dev',
   TIME_BETWEEN_QUESTIONS: 2,
   LINES_PER_CHECK_FORM: 25,
+  // AUTO_MARK: true | false - Automatically mark the check data when we receive it: boolean
+  AUTO_MARK: process.env.hasOwnProperty('AUTO_MARK') ? toBool(process.env.AUTO_MARK) : true,
```

Comment:
i would _prefer_ to move away from the shouty `ENV_VAR` format and take the approach further down the file of the newer items.  `config.AutoMarkCompletedChecks` ?

Reply:
I am happy to change the config object property name, but less happy to make actual environment variables camelCase.  Unix has a widely held convention that exported environment names are upper case. 

Uppercase env vars are ugly; I thought it was a feature.

Would this work for you?

`autoMark: process.env.hasOwnProperty('AUTO_MARK') ? toBool(process.env.AUTO_MARK) : true,`